### PR TITLE
[SCIP] up bugfix release

### PR DESCRIPTION
**Pull request by matbesancon**
Up bugfix release
Add possible concurrent mode
Versioning to follow Ipopt

### CODE REVIEW

File: S/SCIP/build_tarballs.jl [hunk 1]
```diff
@@ -25,6 +25,7 @@ cmake -DCMAKE_INSTALL_PREFIX=$prefix\
   -DAMPL=0\
   -DBOOST=off\
   -DSYM=bliss\
+  -TPI=tny\
```
Comment (matbesancon) [inline S/SCIP/build_tarballs.jl:28]:
```suggestion
  -DTPI=tny\
```

[the remark this answers is not in the document]
Comment (matbesancon) [reply S/SCIP/build_tarballs.jl]: this allows a concurrent mode using tiny threads. I just make naive CMake mistakes and forgot the -D :upside_down_face:

[the remark this answers is not in the document]
Comment (giordano) [reply S/SCIP/build_tarballs.jl]: Oh, probably `-T` is a cmake flag which confused the configuration

**Comment by giordano** [comment]
`clock_gettime` requires linking to librt `-lrt` with old glibc, such as the one we use for x86_64 and i686

**Comment by matbesancon** [comment]
this requires a modification in the cmake itself right?

**Comment by giordano** [comment]
Ideally yes, otherwise something like
```sh
if [[ "${target}" == *86*-linux-gnu ]]; then
   export LDFLAGS="-lrt"
fi
```
before calling CMake should do the trick (ideally with a comment to explain why that's needed

**Pull request merged**